activity.py

```diff
@@ -60,7 +60,7 @@ def build_toolbar(self):
         stop_button = StopButton(self)
         toolbar_box.toolbar.insert(stop_button, -1)
         stop_button.show()
-
+        toolbar_box.show()
```

Comment:
Why is this moved?

Reply:
It did not change anything actually. I was playing around with it. This change won't make any difference.

Reply:
Okay, thanks.  Are you going to remove this part of the change or would you like someone else to do it?

Reply:
Sorry for the delay in response, I had exams. I am working on it currently.